src/main/java/baritone/pathing/calc/AbstractNodeCostSearch.java

```diff
@@ -216,6 +217,8 @@ protected PathNode getNodeAtPosition(int x, int y, int z, long hashCode) {
         if (logInfo) {
             logDebug("Even with a cost coefficient of " + COEFFICIENTS[COEFFICIENTS.length - 1] + ", I couldn't get more than " + Math.sqrt(bestDist) + " blocks");
             logDebug("No path found =(");
+            if (Baritone.settings().desktopNotifications.value)
+                NotificationHelper.notify("No path found =(", true);
```

Comment:
this is on the pathing thread, which i... THINK? is okay? but still a little worrisome.

maybe add a comment into notificationhelper saying "this is not called from the main game thread. do not refer to any minecraft classes, it wouldn't be thread safe"